--- origsrc/calc-2.12.3.2/Makefile	2008-05-09 15:30:25.000000000 -0500
+++ src/calc-2.12.3.2/Makefile	2008-05-13 09:43:57.578125000 -0500
@@ -1156,6 +1156,47 @@
 #
 endif
 
+#################
+# Cygwin target #
+#################
+
+ifeq ($(target),CYGWIN_NT-5.1)
+#
+BLD_TYPE= calc-static-only
+#
+CC_SHARE= -DPIC
+DEFAULT_LIB_INSTALL_PATH= ${PWD}:/lib:/usr/lib:${LIBDIR}:/usr/local/lib
+LD_SHARE= 
+LIBCALC_SHLIB= -shared "-Wl,--enable-auto-image-base,--out-implib,libcalc.dll.a"
+ifdef ALLOW_CUSTOM
+LIBCUSTCALC_SHLIB= -shared "-Wl,--enable-auto-image-base,--out-implib,libcustcalc.dll.a"
+else
+LIBCUSTCALC_SHLIB=
+endif
+#
+CC_STATIC=
+LD_STATIC=
+LIBCALC_STATIC=
+LIBCUSTCALC_STATIC=
+#
+CCWARN= -Wall -W -Wno-comment
+CCWERR=
+CCOPT= ${DEBUG}
+CCMISC=
+#
+LCC= gcc
+CC= ${PURIFY} ${LCC} ${CCWERR}
+EXE= .exe
+#
+LIB_EXT= .dll
+LIB_EXT_VERSION= -${VERSION}${LIB_EXT}
+LIB_EXT_VERS= -${VERS}${LIB_EXT}
+LIB_EXT_VER= -${VER}${LIB_EXT}
+LIB_EXT_VE= -${VE}${LIB_EXT}
+#
+endif
+
+
 #################################
 # Apple MacOS X / Darwin target #
 #################################
